bridge/client/app/_components/ktb-stage-overview/ktb-stage-overview.component.ts

```diff
@@ -114,9 +122,12 @@ export class KtbStageOverviewComponent implements OnInit, OnDestroy {
     this._changeDetectorRef.markForCheck();
   }
 
-  public filterChanged(event: any) {
+  // tslint:disable-next-line:no-any
+  public filterChanged(event: DtFilterFieldChangeEvent<any>) {
```

Comment:
I can't define a custom type because then I receive an error `Argument type ... is not assignable to parameter type ...`, but I'll introduce some casting and new types.